build.zig

```diff
@@ -1,13 +1,13 @@
+const std = @import("std");
+const FileSource = std.build.FileSource;
 const Builder = @import("std").build.Builder;
 
 pub fn build(b: *Builder) void {
-    const mode = b.standardReleaseOptions();
-    const lib = b.addStaticLibrary("ssz.zig", "src/main.zig");
-    lib.setBuildMode(mode);
```

Comment:
`lib.setBuildMode(...)` is gone, and now is part of a configuration struct in `b.addStaticLibrary(...)` (L7).